MainModule/Server/Commands/Creators.lua

```diff
@@ -12,11 +12,10 @@ return function(Vargs, env)
 		DirectBan = {
 			Prefix = Settings.Prefix;
 			Commands = {"directban"};
-			Args = {"username(s)", "reason"};
-			Description = "Adds the specified user(s) to the global ban list; saves";
+			Args = {"username(s)", "reason (optional)"};
```

Comment:
Change `Args = {"username(s)", "reason (optional)"};` to `Args = {"username(s)", "reason?"};`